baybe/campaign.py

```diff
@@ -226,12 +228,13 @@ def add_measurements(
         )
 
         # Telemetry
+        # TODO: Code is inefficient because of unnecessary second fuzzy matching
```

Comment:
Which unnecessary second fuzzy matching are you talking about here? Is this something happening within one of the functions?